src/databricks/labs/ucx/source_code/queries.py

```diff
@@ -1,53 +1,104 @@
 import logging
 from collections.abc import Iterable
-from dataclasses import asdict
+from dataclasses import asdict, dataclass
 from datetime import datetime
 
 from databricks.sdk import WorkspaceClient
 from databricks.sdk.service.sql import ListQueryObjectsResponseQuery
+from databricks.sdk.service.workspace import Language
 
+from databricks.labs.lsql.backends import SqlBackend
 
-from databricks.labs.ucx.source_code.directfs_access import DirectFsAccessCrawler, DirectFsAccessInQuery
+from databricks.labs.ucx.hive_metastore.table_migration_status import TableMigrationIndex
+from databricks.labs.ucx.source_code.base import CurrentSessionState
+from databricks.labs.ucx.source_code.directfs_access import DirectFsAccessCrawler, DirectFsAccessInQuery, LineageAtom
+from databricks.labs.ucx.source_code.linters.context import LinterContext
 from databricks.labs.ucx.source_code.linters.directfs import DirectFsAccessSqlLinter
 
 logger = logging.getLogger(__name__)
 
 
+@dataclass
+class QueryProblem:
+    query_parent: str
+    query_name: str
+    code: str
+    message: str
```

Comment:
can you add lineage of which dashboard it's coming from? without it's pretty much unclear

Reply:
I don't believe we have that information coming from `ws.queries.list()` ?

Reply:
What is the query parent? Is that the dashboard?

Reply:
The parent would be the parent folder, but that is not carried over by the SDK.

Reply:
We're now linting dashboard queries